### ISSUE TITLE
Software problem?

### ISSUE THREAD

**Issue by adelle-l-hay** [SASDigitalHumanitiesTraining/TextEncoding#31]
Hello,

I'm having a problem with applying the transformations. I followed the videos yesterday using the generic Oxygen transformation that Christopher talked about in the "Running XSLT in Oxygen" video and that worked, I got the outputs I was expecting. 

When I try to use transformer.xsl, there is an output, but it is one block of plain text with no formatting. Using the xslt debugger, both Dawn-1-1-1.xml and transformer.xsl are valid, but trying to run the transformation gives this error: "Your stylesheet is version 2, but you chose an XSLT processor that only supports version 1.0. It is recommended that you choose the Saxon-HE/PE/EE processor in the Debugger Engine tool bar."

In the videos, when creating a new transformation scenario, the selected transformer is Saxon-PE 9.8.0.12. When I make a new transformation scenario I only have the option of selecting Saxon-PE 9.9.1.7.

I tried to find a fix for this online but quickly got lost!

**Comment by OlgaBeloborodova** [comment]
Hi Adelle,

Thank you so much for reacting! I, too, have a plain html file in the directory, with no formatting (it doesn't pop up but it is there). Same here with the Saxon version: mine is the same as yours. Might this be the problem?
All best, Olga

**Comment by OlgaBeloborodova** [comment]
Or could it be that we have to enhance the output by adding xsl templates, as per Exercise 1 for today? Perhaps the plain text is precisely what we supposed to produce after the basic transformation?

**Comment by gabrielbodard** [comment]
Yes, if you just run `Transformer.xsl` on an XML file, you will get a HTML file full of plain text, but there are not yet any templates in there to replace XML elements with HTML elements (or text). You need to add templates in there yourself. Try starting with a simple one to replace TEI `<p>` with HTML `<p>` and then work up from there…

**Comment by cmohge1** [comment]
> Hello,
> 
> I'm having a problem with applying the transformations. I followed the videos yesterday using the generic Oxygen transformation that Christopher talked about in the "Running XSLT in Oxygen" video and that worked, I got the outputs I was expecting.
> 
> When I try to use transformer.xsl, there is an output, but it is one block of plain text with no formatting. Using the xslt debugger, both Dawn-1-1-1.xml and transformer.xsl are valid, but trying to run the transformation gives this error: "Your stylesheet is version 2, but you chose an XSLT processor that only supports version 1.0. It is recommended that you choose the Saxon-HE/PE/EE processor in the Debugger Engine tool bar."
> 
> In the videos, when creating a new transformation scenario, the selected transformer is Saxon-PE 9.8.0.12. When I make a new transformation scenario I only have the option of selecting Saxon-PE 9.9.1.7.
> 
> I tried to find a fix for this online but quickly got lost!

Thanks Adelle for this question. I have seen this issue come up before; what I was told by someone who understood this better than me was to select a Saxon-HE parser with the most current version in your software. PE is a 'professional edition' parser so that might throw up errors if you're using a trial software license. HE is the 'home edition' and therefore more applicable for wider use. See https://www.oxygenxml.com/doc/versions/24.1/ug-editor/topics/advanced-saxon-xslt-options.html
